lib/puppet/functions/mysql/innobackupex_args.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
```

Comment:
can we put some description about this function which will help newbie to onboard faster?

Reply:
yes sure